### PR TITLE
created is_prime function to check if number is prime or not 

### PR DESCRIPTION
**Pull request by akhil-gun**

### CODE REVIEW

File: participants/Gunessee/pr_tutorial/simple_functions.py [hunk 1]
```diff
@@ -10,3 +10,13 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+    
+def is_prime(value):
```
Comment (johnedmartz) [inline participants/Gunessee/pr_tutorial/simple_functions.py:14]: Missing docstring but well written.

File: participants/Gunessee/pr_tutorial/simple_functions.py [hunk 2]
```diff
@@ -10,3 +10,10 @@ def factorial(value):
...
+    for i in range(2,value):
```
Comment (olgaborodina) [inline participants/Gunessee/pr_tutorial/simple_functions.py]: great idea to add this function! consider testing if the user accidentally uses a string as an input:)

**Review by johnedmartz — APPROVED** [review]
Well written function.

**Review by olgaborodina — APPROVED** [review]
great job!

File: participants/Gunessee/pr_tutorial/simple_functions.py (hunk 1 above)
Comment (akhil-gun) [reply participants/Gunessee/pr_tutorial/simple_functions.py:14]: Fixed, thank you

File: participants/Gunessee/pr_tutorial/simple_functions.py (hunk 2 above)
Comment (akhil-gun) [reply participants/Gunessee/pr_tutorial/simple_functions.py]: Fixed, thank you

**Comment by ceb8** [comment]
Thanks! 😄

**Pull request merged**